packages/nestjs/yarn.lock

```diff
@@ -0,0 +1,772 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
You can remove this file, it was probably generated before you add the package to the workspace.

Reply:
thanks for the hint, removed it